### PR TITLE
Clear function added to IOS

### PR DESCRIPTION
**Pull request by yigit-serin**
Deinit function does not work as expected.
When I show an ad after that I checked ad is loaded with "isLoaded" function.
It turns true. But after show you cannot show this ad again.
I am not sure about the logic please check it.

### CODE REVIEW

**Comment by ratson** [comment]
@yigit-serin Thank you for the PR.
I am not sure about what problem this fix, would you elaborate more with some code?
Did you follow https://admob-plus.github.io/docs/cordova/ads/interstitial#usage to re-load the ad?

**Comment by yigit-serin** [comment]
@ratson I use time limit to show ads. When I call the load function,  on "admob.interstitial.dismiss" event, I got an exception "Frequency cap reached" I can not load any ad any more.
 I added some triggers to get new ad, I checked is loaded returns true. But I cannot show, because there is no new ad and show function don't throw any exception like Android.

`function click(){
    try {
      await this.interstitial.show();
    } catch (err) {
      this.interstitial.load();
    }
}`

This is my function to some trigger actions. If an ad loaded show method runs. If ad not loaded, show gives and exception and catch loads a new ad and kept in background to show when the "click" method triggered.
It fully works as expected on Android. The problem occurs on IOS.

File: packages/cordova/src/ios/AMBInterstitial.swift [hunk 1]
```diff
@@ -40,17 +43,26 @@ class AMBInterstitial: AMBAdBase, GADFullScreenContentDelegate {
...
+        }
+    }
 
     func adDidRecordImpression(_ ad: GADFullScreenPresentingAd) {
         self.emit(AMBEvents.interstitialImpression)
     }
 
     func ad(_ ad: GADFullScreenPresentingAd, didFailToPresentFullScreenContentWithError error: Error) {
         self.emit(AMBEvents.interstitialShowFail, error)
+        self.clear();
```
Comment (ratson) [inline packages/cordova/src/ios/AMBInterstitial.swift:60]: @yigit-serin Would you move this line above `self.emit` so it aligns with Android implement?

File: packages/cordova/src/ios/AMBInterstitial.swift [hunk 2]
```diff
@@ -40,17 +43,26 @@ class AMBInterstitial: AMBAdBase, GADFullScreenContentDelegate {
...
     }
 
     func adDidPresentFullScreenContent(_ ad: GADFullScreenPresentingAd) {
         self.emit(AMBEvents.interstitialShow)
+        self.clear();
```
Comment (ratson) [inline packages/cordova/src/ios/AMBInterstitial.swift:65]: @yigit-serin Would you move this line above `self.emit` so it aligns with Android implement?

File: packages/cordova/src/ios/AMBInterstitial.swift (hunk 1 above)
Comment (yigit-serin) [reply packages/cordova/src/ios/AMBInterstitial.swift]: I hope it, it is okay?

**Comment by yigit-serin** [comment]
@ratson I hope, it is okay?

**Comment by ratson** [comment]
@yigit-serin Awesome 👍

**Pull request merged**